copier/main.py

```diff
@@ -45,7 +46,7 @@
     from backports.cached_property import cached_property
 
 
-@dataclass
+@dataclass(config=ConfigDict(extra=Extra.forbid))
```

Comment:
Oh, so **this** is how you configure `pydantic.dataclass`!! I must've overlooked this. Some of the documentation and examples were confusing and maybe incorrect for `BaseModel` vs `dataclass`.